osu.Game/Overlays/Options/OptionsSubsection.cs

```diff
@@ -28,7 +29,7 @@ public OptionsSubsection()
                     Margin = new MarginPadding { Bottom = 10 },
                     Font = @"Exo2.0-Black",
                 },
-                Content = new FillFlowContainer

```

Comment:
In IL, it equals to a readonly field with a wrapper property. What you're doing equals to just removing the `readonly`.
You may add 'readonly` to your explicit backing field and try if it works.